<launch>

  <!-- start the node "move_camera" which provides the service "start_move_camera" -->
  <node name="move_camera" pkg="motion_control" type="move_camera" respawn="true" output="screen">
  </node>

  <!-- start the node "rough_localizer" which provides the services "start_rough_localizer" and "stop_rough_localizer" -->
  <node name="rough_localizer" pkg="object_localizer" type="rough_localizer" respawn="true" output="screen">
  </node>

  <!-- start the node "box_segmenter" which provides the services "start_box_segmenter" and "stop_box_segmenter" -->
  <node name="box_segmenter" pkg="object_localizer" type="box_segmenter" respawn="true" output="screen">
  </node>

  <!-- start the node "scan_planner" which provides the service "start_scan_planner" -->
  <node name="scan_planner" pkg="object_localizer" type="scan_planner" respawn="true" output="screen">
    <param name="scan_plan_file" type="str" value="scan_plan.cfg"/>
  </node>

  <!-- start the node "do_scan" which provides the service "start_do_scan" -->
  <node name="do_scan" pkg="motion_control" type="do_scan" respawn="true" output="screen">
    <param name="scan_plan_file" type="str" value="scan_plan.cfg"/>
    <param name="scan_idx" type="int" value="1"/>
  </node>

  <!-- start the node "profile_merger_thread" which provides the service "start_profile_merger" and "stop_profile_merger" -->
  <node name="profile_merger_thread" pkg="object_localizer" type="profile_merger_thread" respawn="true" output="screen">
  </node>

  <!-- start the node "rivet_localizer" which provides the service "start_rivet_localizer" -->
  <node name="rivet_localizer_new" pkg="object_localizer" type="rivet_localizer_new" respawn="true" output="screen">
  </node>

  <!-- start the node "point_rivet" which provides the service "start_point_rivet" -->
  <node name="point_rivet" pkg="motion_control" type="point_rivet" respawn="true" output="screen">
  </node>

</launch>
